File: DotNet/Framework/AutomationFramework/Tests.BDD/StepDefinitions/LoginPageSteps/UserLoginTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.BDD.StepDefinitions.LoginPageSteps
{
    [Binding]
    public class UserLoginTest
    {
        LoginPage Login_Page;
        DashboardPage Dashboard_Page;
        bool isLogoPresent;

        [Given(@"I navigated to OrangeHRM login")]
        public void GivenINavigatedToOrangeHRMPortal()
        {
            Login_Page = LoginPage.Login_Page;
        }

        [When(@"Enter the Username and Password in the login form (.*) and (.*)")]
        public void WhenEnterTheUsernameAndPasswordInTheLoginFormAdminAndAdmin(string username, string password)
        {
            Dashboard_Page = Login_Page.EnterUsername(username)
                .EnterPassword(password)
                .ClickLogin();
        }

        [Then(@"The logo should be present in the Dashboard page")]
        public void ThenTheLogoShouldBePresentInTheDashboardPage()
        {
            Dashboard_Page.IsLogoPresent(out bool isLogoPresent);

            Assert.That(isLogoPresent, Is.EqualTo(true));
        }
    }
}

File: Java/AutomationFramework/src/com/app/BDDTests/LoginPageTests/step_definitions/LoginPageSteps/UserLoginTest.java
package com.app.BDDTests.LoginPageTests.step_definitions.LoginPageSteps;

import com.app.pages.DashboardPage;
import com.app.pages.LoginPage;
import io.cucumber.java.en.Given;
import io.cucumber.java.en.Then;
import io.cucumber.java.en.When;
import org.testng.Assert;

public class UserLoginTest {
    LoginPage Login_Page;
    DashboardPage Dashboard_Page;
    boolean isLogoPresent;
    @Given("I navigated to OrangeHRM login")
    public void iNavigatedToOrangeHRMLogin() {
        Login_Page = LoginPage.GetLoginPage()
                .Init_Elements();
    }

    @When("Enter the Username and Password in the login form {string} and {string}")
    public void enterTheUsernameAndPasswordInTheLoginFormUsernameAndPassword(String username, String password) {
        Dashboard_Page = Login_Page.EnterUsername(username)
                .EnterPassword(password)
                .ClickLogin();
    }

    @Then("The logo should be present in the Dashboard page")
    public void theLogoShouldBePresentInTheDashboardPage() {
        isLogoPresent = Dashboard_Page.IsLogoPresent();

        Assert.assertEquals(isLogoPresent, true);
    }
}
